components/input/input.vue

```diff
@@ -2,6 +2,7 @@
   <div
     ref="container"
     class="base-input"
+    :class="{ 'd-vi-hidden': hidden }"
```

Comment:
Generally not great practice to have multiple class attributes. Preferable to be like this:

```
:class="['base-input', { 'd-vi-hidden': hidden }]"
```